manage-data/lifecycle/data-stream.md

```diff
@@ -22,6 +22,15 @@ To achieve that, it supports:
 
 A data stream lifecycle also supports downsampling the data stream backing indices. See [the downsampling example](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-data-lifecycle) for more details.
 
+## Availability
+
+Note the availability of data stream lifecycle to ensure that it's applicable for your use case:
+
+* Data stream lifecycle is supported only for data streams and cannot be used with standalone indices.
+
+* Like [{{ilm-cap}} ({{ilm-init}})](/manage-data/lifecycle/index-lifecycle-management.md), data stream lifecycle is supported in both {{ecloud}} and self-managed environments. Data stream lifecycle is less feature rich than {{ilm-init}} and is focused on simplicity. For a detailed comparison refer to [Data lifecycle](/manage-data/lifecycle.md).
```

Comment:
I'd avoid "{{ecloud}} and self-managed environments" because we've found that this is not comprehensive enough. to external users, "elastic cloud" could be "elastic cloud on kubernetes", which is technically a self-managed option. I'd indicate either that this is for deployments of the versioned stack, or list out all of the supported deployment types.